csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -294,6 +310,16 @@ public static void ForEach<T>(this IList<T> list, Action<T> action)
             }
         }
 
+        /// <summary>
+        /// <para>Generates a hash code for each item in the list.</para>
```

Comment:
`Generates a hash code for the entire list based on the values of its elements.`